listeners.go

```diff
@@ -329,7 +329,7 @@ func IsUnixNetwork(netw string) bool {
 // include write perms (e.g. `0422` or `022`).
 // Symbolic permission representation (e.g. `u=w,g=w,o=w`)
 // is not supported and will throw an error for now!
-func splitUnixSocketPermissionsBits(addr string) (path string, fileMode fs.FileMode, err error) {
+func SplitUnixSocketPermissionsBits(addr string) (path string, fileMode fs.FileMode, err error) {
```

Comment:
Should this be moved into the `internal` package?

Reply:
That's probably a good idea :+1: